qemu/tests/block_hotplug.py

```diff
@@ -126,7 +124,7 @@ def unplug_device(vm, get_disk_cmd, device):
                 drive.set_param("file", find_image(img_list[num + 1]))
```

Comment:
Not related to this patch but there is a reason the `virttest.qemu_devices.QContainer.images_define_by_variables` was developed and that is to automatically create controllers and detect proper device. The `QRHDrive` is only supported on RHEL but other systems can still use the `QHPDrive`...

Reply:
@ldoktor Yes, I will contact the owner of this feature in our group to see if this can be refactored. Thanks.

Reply:
@vivianQizhu actually a guy from Oracle already noticed that and tried to add QHPDrive support. I replied with `images_define_by_variables` proof-of-concept here: https://github.com/autotest/tp-qemu/pull/1384#pullrequestreview-121494572 it just needs some polishing and a bit of testing, but in general works.

Reply:
@ldoktor Great, we will follow up it, thanks.